cloud-shell/cloudshell_open_cp.sh

```diff
@@ -83,5 +83,11 @@ function cloudshell_open {
       try_envset "$line"
     done <<< "$output"
   fi
+  # add sandboxctl to path
+  export PATH="~/cloudshell_open/cloud-ops-sandbox/sre-recipes:$PATH"
+  export PATH="~/cloudshell_open/stackdriver-sandbox/sre-recipes:$PATH"
```

Comment:
Do we need to leave a comment letting people know this is for previous releases?

Reply:
good point, added